src/front/zsharp/mod.rs

```diff
@@ -305,6 +305,24 @@ impl<'ast> ZGen<'ast> {
                     Ok(uint_lit(DFL_T.modulus().significant_bits(), 32))
                 }
             }
+            "vector_add" => {
```

Comment:
I think this name should specify the type it handles, because it's not generic.

Maybe vadd_u32 or vec_add_u32 or addvec_u32?